console/console-init/ui/src/modules/address/components/AddressConfiguration/AddressConfiguration.tsx

```diff
@@ -155,14 +155,17 @@ const AddressConfiguration: React.FunctionComponent<IAddressConfigurationProps>
                 />
               </FormGroup>
             )}
+            {(type?.toLocaleLowerCase() === "topic" ||
```

Comment:
Please use enum type `AddressTypes` for values "queue", "topic" and "subscription"